snappi_ixnetwork/ngpf.py

```diff
@@ -80,6 +87,11 @@ def _config_proto_stack(self, ixn_obj, snappi_obj, ixn_dg):
             stack_class = getattr(self, '_configure_{0}'
```

Comment:
add type check for container_name and name, then raise exception

Reply:
According to plan must filed should validate in Model layer. Anyway I have added check for container_name.